demo/perf/src/main/java/org/apache/servicecomb/demo/perf/PerfMetricsFilePublisher.java

```diff
@@ -75,51 +70,57 @@ private void collectVertxMetrics(MetricsLoader loader, StringBuilder sb) {
   }
 
   private void collectMetrics(MetricsLoader loader, StringBuilder sb) {
-    MetricNode treeNode = loader
-        .getMetricTree(MetricsConst.SERVICECOMB_INVOCATION, MetricsConst.TAG_ROLE, MetricsConst.TAG_OPERATION,
-            MetricsConst.TAG_STATUS);
+    MetricNode treeNode;
+    try {
+      treeNode = loader
+          .getMetricTree(MetricsConst.SERVICECOMB_INVOCATION, MetricsConst.TAG_ROLE, MetricsConst.TAG_OPERATION,
+              MetricsConst.TAG_STATUS);
+    }
+    //before receive any request,there are no MetricsConst.SERVICECOMB_INVOCATION,so getMetricTree will throw ServiceCombException
+    catch (ServiceCombException ignored) {
+      return;
```

Comment:
We need to log the exception for trace the issue

Reply:
add a containsId method and no need throw exception any more